unit-tests/func/reflectivity/l500/test-sanity.cpp

```diff
@@ -0,0 +1,64 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2021 Intel Corporation. All Rights Reserved.
+
+//#test:tag ir-reflectivity
+//#test:device L500*
+
+#include "../../func-common.h"
+
+// Test group description:
+//       * This tests group verifies IR reflectivity option restrictions,
+//         according to [RS-8358].
+//
+// Current test description:
+//       * All conditions for IR Reflectivity set, verify enabling option succeed.
+//       * All conditions for IR Reflectivity set and depth sensor is streaming, verify enabling option succeed.
+//
+// Note: * L515 specific test.
+//       * Test will pass if no L515 device is connected.
+//       * Test will be performed on the first L515 device detected.
+
+TEST_CASE( "Not streaming - sanity", "[reflectivity]" )
```

Comment:
Don't we do this as part of the other tests?

Reply:
I believe that it is covered somehow , but if the main flow will break in the future you want to investigate the simples test to see what is wrong instead of going to the complicated ones,
The unit tests starts from sanity and then gets complicated with all other requirements checks.
I like simplicity.

Reply:
OK, your decision